templates/cloudflare-workers/server.ts

```diff
@@ -1,7 +1,53 @@
-import { createEventHandler } from "@remix-run/cloudflare-workers";
+import { getAssetFromKV } from "@cloudflare/kv-asset-handler";
+import type { AppLoadContext } from "@remix-run/cloudflare";
+import { createRequestHandler, logDevReady } from "@remix-run/cloudflare";
 import * as build from "@remix-run/dev/server-build";
+import manifestJSON from "__STATIC_CONTENT_MANIFEST";
```

Comment:
I don’t think this line will work with `serverDependenciesToBundle: "all"`, unless something has changed recently. The bundler should be complaining about being unable to find this dependency. _If_ that’s the case, the solution is to use `serverDependenciesToBundle: [/^(?!(__STATIC_CONTENT_MANIFEST)$).*$/u]`, which is hacky but at least avoids the issue.